.github/workflows/cmake_builds.yml

```diff
@@ -235,18 +236,23 @@ jobs:
             -DCMAKE_INSTALL_PREFIX=$PWD/install-gdal \
             "-DCMAKE_UNITY_BUILD=${CMAKE_UNITY_BUILD}" \
             -DGDAL_USE_MYSQL:BOOL=OFF \
-            -DBUILD_PYTHON_BINDINGS:BOOL=OFF \
             -DBUILD_CSHARP_BINDINGS=ON
         working-directory: ${{ github.workspace }}
       - name: Build
         run: cmake --build build -j 3
         working-directory: ${{ github.workspace }}
-      # FIXME: enable some tests
-      #- name: test
-      #  run: cmake --build build --target quicktest
-      #  working-directory: ${{ github.workspace }}
-      #  env:
-      #    PROJ_LIB: "C:\\tools\\msys64\\mingw64\\share\\proj"
+      - name: test
+        run: cmake --build build --target quicktest
+        working-directory: ${{ github.workspace }}
+      - name: test (with ctest)
+        shell: bash -l {0}
```

Comment:
... and this will use a different `bash` and msys runtime than the other commands (which run in a msys2 wrapper). I'm concerned that this may be fragile.

Reply:
> I'm concerned that this may be fragile.

for the git part, this was necessary otherwise there were wrong line endings (I suppose that setting the options of the git under the msys2 shell had no effect on the actions/checkout@v2)
for the tests, this is indeed not intended / a copy&paste from another config. I'll issue a PR to remove it